guides/common/modules/con_about-running-jobs-on-hosts.adoc

```diff
@@ -11,6 +11,12 @@ Communication occurs through {SmartProxyServer}, which means that {ProjectServer
 Remote execution uses the SSH service that must be enabled and running on the target host.
 Ensure that the remote execution {SmartProxy} has access to port 22 on the target hosts.
 
+Enable and configure the `pull-mqtt` pull mode on the {SmartProxy}.
+When using `pull-mqtt` for pull mode, the remote client initiates all connections, and there are no outgoing connections from the proxy.
+{Project} delegates pull jobs to the proxy, and the proxy notifies the host of a new job over the Message Queueing Telemetry Transport (MQQT).
```

Comment:
```suggestion
{Project} delegates pull jobs to the proxy, and it notifies the host of a new job over the Message Queueing Telemetry Transport (MQQT).
```
Minor observation.

Reply:
@mjivraja I would keep there the proxy. There is one technical writing rule. Don't use "it". We always specify the subject even if we need to use it repeatedly, therefore, Sarah's version is better.

Reply:
> 

Noted. :+1: @vferschm